assets/js/components/KeyMetrics/ChipTabGroup/index.js

```diff
@@ -259,20 +326,32 @@ export default function ChipTabGroup( { allMetricItems, savedItemSlugs } ) {
 										({ selectedCounts[ group.SLUG ] })
 									</span>
 								) }
+								{ !! newlyDetectedMetrics?.[ group.SLUG ] && (
+									<span className="googlesitekit-chip-tab-group__chip-item-new-dot" />
+								) }
 							</Tab>
 						) ) }
 					</TabBar>
 				) }
 			</div>
 			<div className="googlesitekit-chip-tab-group__tab-item">
-				{ Object.keys( activeMetricItems ).map( ( slug ) => (
-					<MetricItem
-						key={ slug }
-						slug={ slug }
-						savedItemSlugs={ savedItemSlugs }
-						{ ...activeMetricItems[ slug ] }
-					/>
-				) ) }
+				{ Object.keys( activeMetricItems ).map( ( slug ) => {
+					const metricGroup = activeMetricItems[ slug ].group;
+					const isNewlyDetected =
+						newlyDetectedMetrics?.[ metricGroup ]?.includes(
+							slug
+						) ?? false;
```

Comment:
Same here re: nullish coalescing.
```suggestion
						newlyDetectedMetrics?.[ metricGroup ]?.includes(
							slug
						);
```